mods/d2k/rules/structures.yaml

```diff
@@ -838,6 +838,7 @@ high_tech_factory:
 		UnitType: ornithopter
 		DisplayBeacon: True
 		CameraActor: camera
+		ClockSequence:
```

Comment:
Can we please fix the default on the trait instead?  Having RA and TD specifically define `beacon` makes more sense than D2K and TD overriding it back to nothing.